docs/section5/Rule5-33.md

```diff
@@ -0,0 +1,43 @@
+
+# Envelope - Rule 5-33  
+
+**Rule ID:** 5-33  
+**Rule Description:** Automatically controlled fenestration shading devices must be modeled in the proposed design the same as in user model.  
+**Rule Assertion:** P-RMR subsurface:has_automatic_shades = U-RMR subsurface:has_automatic_shades  
+**Appendix G Section:** Section G3.1-5(a)(4) Building Modeling Requirements for the Proposed design  
+**Appendix G Section Reference:**  None
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** None  
+
+**Manual Checks:** None  
+**Evaluation Context:**  Each Data Element  
+**Data Lookup:** None  
+**Function Call:**  
+
+  1. match_data_element()
+  2. get_opaque_surface_type()
+
+## Rule Logic:
+
+- For each building segment in the Proposed model: `for building_segment_p in P_RMR.building.building_segments:`
+
+  - For each thermal block in building segment: `for thermal_block_p in building_segment_p.thermal_blocks:`
+
+    - For each zone in thermal block: `for zone_p in thermal_block_p.zones:`
+
+      - For each surface in zone: `for surface_p in zone_p.surfaces:`
+
+        - Check if surface is above-grade wall: `if get_opaque_surface_type(surface_p) == "ABOVE-GRADE WALL":`
+
```

Comment:
Don't need this check

Reply:
P RMR should have same number of sub-surfaces as U-RMR

Reply:
Wall check and the function is now removed.